landsea_mask/README.md

```diff
@@ -0,0 +1,26 @@
+# Land-sea mask (from ETOPO2022)
+
+## Overview
+
+This artifact provides a land-sea mask computed from the 30 arc-second ETOPO2022
+orography information.
+- `landsea_mask.nc` (30 arc-second, compressed, 2MB)
+
+This mask is generated by identifying connected basins below a predefined
+sea-level (set as 0 in this artifact) and marking them as "land" if they smaller
+than a given size. The size is provided as fraction of the surface of the globe
+(where the ocean is 65 %). This artifact has all the lakes filled, so that it
+identifies as land everything that is not ocean. Generating this mask is
+memory-intensive: for the 30 arcsecond dataset, it requires 50 GB of RAM.
+
+The name of the variable in the dataset is `landsea` and the data type is an
+Int8: 0 is ocean, 1 is land. In the future, we might include lakes as well.
+
+Note also that the data is compressed. If this leads to problems, please open an
+issue. Uncompressed data requires 400x more space.
+
+## References
+
+NOAA National Centers for Environmental Information. 2022: ETOPO 2022 15 Arc-Second
```

Comment:
```suggestion
NOAA National Centers for Environmental Information. 2022: ETOPO 2022 30 Arc-Second
```

Reply:
I wonder if this is a typo ; the other docstrings and README suggest that the source is the ETOPO2022 30 arc-second datasets.

Reply:
Seems like NCEI has some system outages right now ; this citation otherwise seems consistent with the recommendation on their dataset landing page. Closing this conversation as resolved.

Reply:
Yes, I noticed that too. 

I fixed the 15->30